### PR TITLE
Add cJSON_Utils as an option

### LINKED ISSUE #3559: [package] cjson/1.7.14: Does not copy cJSON_Utils.h to the include directory.
### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **cjson/1.7.14**
  * Operating System+version: **macOS 11.0.1**
  * Compiler+version: **Apple clang version 12.0.0 (clang-1200.0.32.21)**
  * Conan version: **conan 1.32.0-dev**
  * Python version: **Python 3.8.6**

### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```
Configuration for profile default:

[settings]
os=Macos
os_build=Macos
arch=x86_64
arch_build=x86_64
compiler=apple-clang
compiler.version=12.0
compiler.libcxx=libc++
build_type=Release
[options]
[build_requires]
[env]
```

### Steps to reproduce (Include if Applicable)

Add cJSON to a C project with a Conan file:

```
[build_requires]
cjson/1.7.14

[generators]
cmake_find_package
cmake_paths

[options]
cjson:shared=True
```

Then try to use `cjson/cJSON.h` and `cjson/cJSON_Utils.h`:

```c
#include "cjson/cJSON.h"
#include "cjson/cJSON_Utils.h" /* for sorting */

/* Some code here that uses it */
```

Find that Conan has lost half of the files:

```
fatal error: 'cjson/cJSON_Utils.h' file not found
```

Which is surprising because both files exist in `source_subfolder`:

```
./source/source_subfolder/cJSON.h
./source/source_subfolder/cJSON_Utils.h
```

Yet, only `cJSON.h` is copied to the include directories. Looks like a bug to me.

### PR DESCRIPTION
**Pull request by dertuxmalwieder**
Trying to fix #3559

Specify library name and version:  **cjson/1.7.14**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/cjson/all/conanfile.py [hunk 1]
```diff
@@ -15,11 +15,13 @@ class CjsonConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "build_utils": [True, False],
```
Comment (prince-chrismc) [inline recipes/cjson/all/conanfile.py:18]: The _unofficial_ convention is to have `with_` but it's not a hard requirement!

**Review by prince-chrismc — COMMENTED** [review]
https://github.com/conan-io/conan-center-index/blob/c79a87606e2affdde7969a1c008d4e33a474b321/recipes/cjson/all/conanfile.py#L93

I am not sure it this will be a problem with the link libs order 🤔

File: recipes/cjson/all/conanfile.py (hunk 1 above)
Comment (dertuxmalwieder) [reply recipes/cjson/all/conanfile.py:18]: Technically, `with_` should be used for `fPIC` as well... but of course, I can change that. No need to violate *unofficial* conventions with the first try. ;-)

**Comment by prince-chrismc** [comment]
> Sorry, the build is only launched for Early Access Program users. You can request access [writing in this issue](https://github.com/conan-io/conan-center-index/issues/4).

@dertuxmalwieder make sure to request access!

**Review by prince-chrismc — APPROVED** [review]

**Comment by dertuxmalwieder** [comment]
Ahh... that happens when I do a dozen things at the same time.
Requested!

**Comment by SpaceIm** [comment]
Sorry I missed your PR.
https://github.com/conan-io/conan-center-index/pull/3581

**Comment by dertuxmalwieder** [comment]
Too many bots here. 😂 ...
No problem @SpaceIm :-)

**Comment by dertuxmalwieder** [comment]
Ah - fixed. Good, thank you!

**Pull request closed without merging**